manim/_config/utils.py

```diff
@@ -737,6 +738,13 @@ def digest_file(self, filename: str) -> "ManimConfig":
         multiple times.
 
         """
+        if not os.path.isfile(filename):
+            raise FileNotFoundError(
+                errno.ENOENT,
```

Comment:
Is it required to add `errno.ENOENT`?

Reply:
@naveen521kk It feels like a proper way to do it, but hardcoding this particular error number (2) will likely work just as well.